### PR TITLE
Remove branch matching for atlas

### PR DESCRIPTION
**Pull request by guillaumevernieres**
Might address #370

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ env:
     - BUILD_OPT_OOPS="-DENABLE_OOPS_TOYMODELS=OFF"
     - BUILD_OPT_UFO="-DLOCAL_PATH_TESTFILES_IODA=NONE"
     - BUILD_OPT_SOCA="-DSOCA_TESTS_FORC_DEFAULT_TOL=ON -DCRTM_FIX_DIR=../../repo.src/crtm/fix"
-    - MATCH_REPOS="atlas oops saber ioda ioda-converters ufo soca soca-config"
+    - MATCH_REPOS="oops saber ioda ioda-converters ufo soca soca-config"
```
Comment (aerorahul) [inline .travis.yml]: saber should come before oops.

**Review by aerorahul — APPROVED** [review]
looks good.

**Comment by guillaumevernieres** [comment]
@aerorahul thanks for the quick review!

**Comment by mer-a-o** [comment]
The order was changed a while ago. oops should go before saber.

**Pull request merged**